mappings/net/minecraft/world/gen/decorator/NopeDecorator.mapping

```diff
@@ -0,0 +1 @@
+CLASS byl net/minecraft/world/gen/decorator/NopeDecorator
```

Comment:
NopeDecorator? What does this do? Void?

Reply:
Not sure, it's ID is literally "nope". I think it's just a default one or something. Couldn't really tell

Reply:
EmptyDecorator?

Reply:
Yeah, Empty, Void, Default?